### PR TITLE
(feat) performanceChart bugFix

### PR DESCRIPTION
**Pull request by anu-aj**
### Functionality:
Due to change in DB, Performance graph is not working

### Solution:
Bug Fixed and Code Updated

### Risk level:
- [ ] high 
- [ ] medium
- [x] low

### How to test:
1. ng serve
2.  firebase emulators:start

### CODE REVIEW

File: src/app/body/my-dashboard/performance-chart/performance-chart.component.ts [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import { Component, Input, OnInit } from '@angular/core';
-import { AngularFirestore } from '@angular/fire/firestore';
-import { Tasks } from 'src/app/Interface/TasksInterface';
+import { AngularFirestore, AngularFirestoreCollectionGroup } from '@angular/fire/firestore';
+import { Observable } from 'rxjs';
+import { Tasks, TasksId } from 'src/app/Interface/TasksInterface';
```
Comment (viv9k) [inline src/app/body/my-dashboard/performance-chart/performance-chart.component.ts:4]: These changes are not required

File: src/app/body/my-dashboard/performance-chart/performance-chart.component.ts [hunk 2]
```diff
@@ -18,7 +20,10 @@ export class PerformanceChartComponent implements OnInit {
   sprintRange1: number
   sprintRange2: number
   data = [];
-  constructor(public db: AngularFirestore, public errorHandlerService: ErrorHandlerService) { }
+  tasksCollection: AngularFirestoreCollectionGroup<Tasks>
+  tasksData: Observable <TasksId[]>
+
```
Comment (viv9k) [inline src/app/body/my-dashboard/performance-chart/performance-chart.component.ts:25]: These changes are also not required

**Review by viv9k — APPROVED** [review]
Please complete the required changes asap.

File: src/app/body/my-dashboard/performance-chart/performance-chart.component.ts (hunk 1 above)
Comment (anu-aj) [reply src/app/body/my-dashboard/performance-chart/performance-chart.component.ts]: okay

File: src/app/body/my-dashboard/performance-chart/performance-chart.component.ts [hunk 3]
```diff
@@ -1,6 +1,9 @@
 import { Component, Input, OnInit } from '@angular/core';
-import { AngularFirestore } from '@angular/fire/firestore';
-import { Tasks } from 'src/app/Interface/TasksInterface';
+import { AngularFirestore, AngularFirestoreCollectionGroup } from '@angular/fire/firestore';
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { Tasks, TasksId } from 'src/app/Interface/TasksInterface';
```
Comment (viv9k) [inline src/app/body/my-dashboard/performance-chart/performance-chart.component.ts:5]: revert these also

**Review by viv9k — CHANGES_REQUESTED** [review]
please complete the required changes

File: src/app/body/my-dashboard/performance-chart/performance-chart.component.ts (hunk 3 above)
Comment (anu-aj) [reply src/app/body/my-dashboard/performance-chart/performance-chart.component.ts:5]: reverted

**Review by viv9k — APPROVED** [review]
ship it

**Pull request merged**